lib/ansible/modules/network/cnos/cnos_lldp.py

```diff
@@ -0,0 +1,205 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+#
+# Copyright (C) 2019 Lenovo.
+# (c) 2017, Ansible by Red Hat, inc
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+#
+# Module to work on Link Aggregation with Lenovo Switches
+# Lenovo Networking
+#
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = """
+---
+module: cnos_lldp
+version_added: "2.8"
+author: "Anil Kumar Muraleedharan (@amuraleedhar)"
+short_description: Manage LLDP configuration on Lenovo CNOS network devices.
+description:
+  - This module provides declarative management of LLDP service
+    on Lenovc CNOS network devices.
+notes:
+  - Tested against CNOS 10.9.1
+options:
+  state:
+    description:
+      - State of the LLDP configuration. If value is I(present) lldp will be
+         enabled else if it is I(absent) it will be disabled.
+    default: present
+    choices: ['present', 'absent']
+  provider:
```

Comment:
We'd like to avoid adding deprecated options to new modules, such as `provider`

Reply:
Done.